fdbserver/masterserver.actor.cpp

```diff
@@ -953,6 +955,7 @@ ACTOR Future<Void> sendInitialCommitToResolvers(Reference<MasterData> self) {
 		wait(yield());
 	}
 	wait(waitForAll(txnReplies));
+	TraceEvent("SentTxnStateStoreToCommitProxies", self->dbgid);
```

Comment:
This sounds too general. I think you want to add word "Recovery" in it since it is only used in recovery.